### ISSUE TITLE
What's New 

### ISSUE THREAD

**Issue by DmitryBespalov** [safe-global/safe-ios#2734]
- Rebranding
  - Title: New look! ✨
  - Text: Safe is following up with a new look. We have updated the color palette in the app, including dark mode.
- NFTs / Collectibles
  - Title: Collectibles are back 💥
  - Text: Now you can view collectibles in your Safe. Access it from the Assets tab - Collectibles.
- Keystone
  - Title: Keystone support 🗝
  - Text: We have partnered with Keystone to support the user-friendly hardware wallet in the app natively. Connect it from Settings - Owner Keys - Add.

Icons: [Figma](https://www.figma.com/file/NynYLm0HPB49Q8tApebw0n/What's-New?node-id=646%3A392)

**Comment by biafra23** [comment]
> We have partnered up with the Keystone to support the user-friendly hardware wallet natively in the app

I would rephrase this to (removing "the" before Keystone): 
> We have partnered up with Keystone to support the user-friendly hardware wallet natively in the app

**Comment by DmitryBespalov** [comment]
Updated wording

**Comment by liliya-soroka** [comment]
The dark view was not tested as it's blocked by - https://app.zenhub.com/workspaces/mobile-engineering-60cc78740f488d000e996335/issues/safe-global/safe-ios/2738

The light theme view is verified
app version 3.18.0 (1410)

**Comment by DmitryBespalov** [comment]
@liliya-soroka you can see the dark theme if you enable it in the phone (system) settings

**Comment by liliya-soroka** [comment]
Verified in dark theme too
app version 3.18.0(1410)
